data/shreyanshs31.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "Shreyansh Shukla",
+  "displayStatsPublic":"true",
+  "bio": "Student👨🏽‍🎓 • DSA👨🏽‍💻 • Python • Java •  Learning in Public👤 • Love Open Source💜"
+  "type": "personal",
+  "avatar":"https://github.com/shreyanshs31.png",
```

Comment:
No need to add an avatar i guess, It automatically fetches profile picture which you use for github
```suggestion
```